package/MDAnalysis/analysis/dihedrals.py

```diff
@@ -99,39 +102,69 @@ class which takes a list of `atomgroups`, this class only needs a list of
    trajectory (XTC). The contours in the background are the "allowed region"
    and the "marginally allowed" regions.
 
-The Janin class works in the same way, only needing a list of residues; see the
-:ref:`Janin plot figure <figure-janin>` as an example. To plot the data
-yourself, the angles can be accessed using :attr:`Ramachandran.angles` or
-:attr:`Janin.angles`.
-
-Reference plots can be added to the axes for both the Ramachandran and Janin
-classes using the kwarg ``ref=True``. The Ramachandran reference data
-(:data:`~MDAnalysis.analysis.data.filenames.Rama_ref`) and Janin reference data
-(:data:`~MDAnalysis.analysis.data.filenames.Janin_ref`) were made using data
-obtained from a large selection of 500 PDB files, and were analyzed using these
-classes. The allowed and marginally allowed regions of the Ramachandran reference
-plt have cutoffs set to include 90% and 99% of the data points, and the Janin
-reference plot has cutoffs for 90% and 98% of the data points. The list of PDB
-files used for the referece plots was taken from [Lovell2003]_ and information
-about general Janin regions was taken from [Janin1978]_.
+To plot the data yourself, the angles can be accessed using
+:attr:`Ramachandran.angles`.
 
 .. Note::
-   These classes are prone to errors if the topology contains duplicate or missing
-   atoms (e.g. atoms with `altloc` or incomplete residues). If the topology has as
-   an `altloc` attribute, you must specify only one `altloc` for the atoms with
-   more than one (``"protein and not altloc B"``).
 
+   The Ramachandran analysis is prone to errors if the topology contains
+   duplicate or missing atoms (e.g. atoms with `altloc` or incomplete
+   residues). If the topology has as an `altloc` attribute, you must specify
+   only one `altloc` for the atoms with more than one (``"protein and not
+   altloc B"``).
+
+
+Janin analysis
+~~~~~~~~~~~~~~
+
+Janin plots [Janin1978]_ for side chain conformations (:math:`\chi_1` and
+:math:`chi_2` angles) can be created with the
+:class:`~MDAnalysis.analysis.dihedrals.Janin` class. It works in the same way,
+only needing a list of residues; see the :ref:`Janin plot figure
+<figure-janin>` as an example.
+
+The data for the angles can be accessed in the attribute or
+ :attr:`Janin.angles`.
```

Comment:
```suggestion
:attr:`Janin.angles`.
```

The space makes this whole line big, like a heading :/